iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java

```diff
@@ -734,6 +734,26 @@ public int getAssignedSeriesPartitionSlotsCount(String database) {
     return databasePartitionTables.get(database).getAssignedSeriesPartitionSlotsCount();
   }
 
+  /**
+   * Only leader use this interface.
+   *
+   * <p>Get the assigned TimePartitionSlots count in the specified Database
+   *
+   * @param database The specified Database
+   * @return The assigned TimePartitionSlots count
+   */
+  public long getAssignedTimePartitionSlotsCount(String database) {
```

Comment:
Instead of calling the existing function to generate the array and then summing it, which can increase gc pressure, I recommend separating out the function to compute a long directly

Reply:
OK